core/src/smartcontracts/isi/permissions/checks.rs

```diff
@@ -297,3 +297,58 @@ pub fn check_query_in_instruction(
         Instruction::Fail(_) | Instruction::ExecuteTrigger(_) => Ok(()),
     }
 }
+
+/// Check if permission `token` has right parameters according to `definition`.
+///
+/// Requires only one bypass over `token` and `definition` parameters.
```

Comment:
Rephrase.

Reply:
I guess you should say that since the parameters are in the alphabetical order, all you need is one iteration, right? Or does this comment means something different?

Reply:
The idea was that at the first look it's not trivial how to make this function in the most efficient and most convinient way. But yes, I mean, that it only needs one iteration